### PR TITLE
Add support for vaultUrl in azure keyvault plugin

### PR DESCRIPTION
**Pull request by anatbal**
Currently, the Azure Key Vault plugin enforces that the URL of the vault must follow the format of "https://%s.vault.azure.net". 
A new configuration parameter called "vaultURL" has been added in this pull request. This new parameter allows for the possibility of other suffixes to be used, thereby enabling support for multiple clouds, such as the government cloud. 
It's worth noting that the "vault" parameter will still be available, but its value will be overridden by the "vaultURL" parameter if exists.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,6 +42,11 @@ The `azure.keyvault` plugin resolves credentials or parameters against secrets i
     vault = "myvault"
     ```
 1. [Create a key vault][keyvault] and set the vault name in the config with name of the vault.
+- If you want to specifiy the full url for your vault, use vault-url instead:
+```toml
+   [secrets.config]
+    vault-url = "https://myvault.vault.azure.net"
+```
```
Comment (carolynvs) [inline README.md:49]: The bullet point renders a bit oddly, so how about we make this a new paragraph within the list item instead (along with a spelling fix and a note about which value takes precedence):

```suggestion
1. [Create a key vault][keyvault] and set the vault name in the config with name of the vault.
   
   If you want to specify the full url for your vault, use vault-url instead. Note that `vault-url` takes precedence over the `vault` setting.
```toml
   [secrets.config]
    vault-url = "https://myvault.vault.azure.net"
```
```

**Review by carolynvs — CHANGES_REQUESTED** [review]
Looks good! I just have one small edit to the docs

**Comment by anatbal** [comment]
@carolynvs Thanks! I fixed the docs with your suggestion

**Review by carolynvs — APPROVED** [review]
Thank you for the fix! 🎉

**Comment by carolynvs** [comment]
Your fix is now available in the v1.2.0 release: https://github.com/getporter/azure-plugins/releases/tag/v1.2.0

**Pull request merged**